site/prod_nginx.conf

```diff
@@ -69,4 +69,4 @@ http {
       return 200 "User-agent: *\nDisallow: /information-and-services/\nDisallow: /information-and-services";    
     }
   }
-}
+}
```

Comment:
@BrendanCNg I think we removed this file from the PR as it could potentially break our deployment configuration.

CSS changes look good though, how does the `-5rem` look on mobile / tablet?